_topic_map.yml

```diff
@@ -238,6 +238,13 @@ Topics:
   - Name: AWS Service Broker
     File: aws_broker
 ---
+Name: Backing up applications and projects
```

Comment:
I know that this is already long, but should this also mention "restoring" in addition to backing up? (Because the assembly handles both backup and restore). You could also consider reducing to something like "Backup and restore" or "Backing up and restoring" or something

Reply:
I went with "Backup and Restore."